.github/ISSUE_TEMPLATE/01.bug_report.yml

```
@@ -0,0 +1,48 @@
+name: バグ報告
+description: バグ報告
```

nameに対してdescriptionはその説明になっているといいのかなと思います。

```suggestion
name: Bug Report
description: バグ・不具合報告を行うテンプレートです。
```